experimental/packages/opentelemetry-configuration/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@opentelemetry/configuration",
+  "version": "0.1",
```

Comment:
We should already have it at the same version as other experimental packages. 🙂 The spec requires us to version in lockstep.